third_party/istio-1.1.2/istio-lean.yaml

```diff
@@ -395,13 +395,6 @@ data:
     configSources:
     - address: istio-galley.istio-system.svc:9901
 
-    # PATCH #3: Adding default connection timeout.
```

Comment:
@tcnghia FYI, this patch should have been deleted in the PR https://github.com/knative/serving/pull/3671